src/components/browse/sidebar/sidebar.tsx

```diff
@@ -89,8 +92,8 @@ const Sidebar = (props: sidebarProps) => {
       <Row className="mb-2 mt-3 justify-content-end">
         <Col>
           <Button
-            className="w-100 rounded border-2"
-            variant="outline-dark"
+            className="w-100 rounded border-2 shadow-sm"
+            variant="outline-secondary"
             onClick={() => {
               handleClear();
               scrollUp();
```

Comment:
scrollUp doesn't go to the top for me, not sure if it is the same behaviour for you as well, please check.

Reply:
We can add as a bug.